guide/english/agile/continuous-delivery/index.md

```diff
@@ -7,5 +7,29 @@ Continuous delivery (CD) is a software engineering approach in which teams produ
 
 It aims at building, testing, and releasing software faster and more frequently. The approach helps reduce the cost, time, and risk of delivering changes by allowing for more incremental updates to applications in production. A straightforward and repeatable deployment process is important for continuous delivery. Continuous delivery means that the team ensures every change can be deployed to production but may choose not to do it, usually due to business reasons.
 
+### The myth
 
+It is often assumed that if we want to deploy software more frequently, we must accept _lower levels of stability and reliability_ in our systems.
+As a matter of fact, peer-reviewed research shows that this is not the case—high performance teams consistently deliver services faster and more reliably than their low performing competition. This capability provides an incredible competitive advantage for organizations, at ultimately a lower cost in efforts.
 
+### The benefits
+
+The practices at the heart of **CD** will help you achieve a few very important benefits:
+
+* **Low Risk releases.** The primary goal of continuous delivery is to make releases/deployments to production much painless, low-risk events that can be performed at any given time, on demand. By applying certain patterns, it is relatively straightforward to achieve zero-downtime deployments that are undetectable to users.
+
+* **Faster time to market.** When teams work togetherto automate the build and deployment. environment provisioning, and regression testign processes, developers can incorporate integration and regression testing into their daily work and completely remove these phases. Therefore, large amounts of re-work that plague the phased approach can be avoided.
```

Comment:
keep space between `togetherto` -> `together to`
grammatical error: `testign` -> `testing`